generated/nirfmxnr/nirfmxnr.proto

```diff
@@ -0,0 +1,57 @@
+
+//---------------------------------------------------------------------
+// This file is automatically generated. All manual edits will be lost.
+//---------------------------------------------------------------------
+// Proto file for the NI-RFMXNR Metadata
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.ni.grpc.nirfmxnr";
+option java_outer_classname = "NiRFmxNR";
+option csharp_namespace = "NationalInstruments.Grpc.NiRFmxNR";
+
+package nirfmxnr_grpc;
+
+import "session.proto";
+
+service NiRFmxNR {
+  rpc Close(CloseRequest) returns (CloseResponse);
+  rpc Initialize(InitializeRequest) returns (InitializeResponse);
+}
+
+enum NiRFmxNRAttribute {
+  NIRFMXNR_ATTRIBUTE_UNSPECIFIED = 0;
+}
+
+enum Boolean {
+  option allow_alias = true;
+  BOOLEAN_UNSPECIFIED = 0;
+  BOOLEAN_FALSE = 0;
+  BOOLEAN_TRUE = 1;
+}
+
+message CloseRequest {
+  nidevice_grpc.Session instrument_handle = 1;
+  oneof force_destroy_enum {
+    Boolean force_destroy = 2;
+    int32 force_destroy_raw = 3;
+  }
+}
+
+message CloseResponse {
+  int32 status = 1;
+}
+
+message InitializeRequest {
+  string session_name = 1;
+  string resource_name = 2;
+  string option_string = 3;
+}
+
+message InitializeResponse {
+  int32 status = 1;
+  nidevice_grpc.Session handle_out = 2;
```

Comment:
I just used the same param names as what was in the C Header. Figured when we update the handcoded metadata with the generated metadata I can fix up the session tests to use whatever parameter names we land on.

Reply:
Yeah, I think we're pretty set on `instrument` for both `handle_out` and `instrument_handle` in. And any other changes would be special cases (bad snake_case conversions mostly).

But you can apply that later.